### PR TITLE
change docs place

### PR DESCRIPTION
**Pull request by Zeroto521**
move docs to https://flow-launcher.github.io/docs

## changelogs

1. format with markdown lint
2. change docs link

### CODE REVIEW

**Comment by taooceros** [comment]
image
the version number seems to be wrong 🤣

**Comment by jjw24** [comment]
we should keep Integration and Usage because we want people to be able to get off the ground running with Flow at a glance of our readme rather than going through docs. For more advanced usage we can keep in docs

**Comment by jjw24** [comment]
i think we should add a badge to our documentation as well

File: README.md [hunk 1]
```diff
@@ -37,26 +38,25 @@ Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at be
...
 
-**Usage**
 - Open flow's search window: <kbd>Alt</kbd>+<kbd>Space</kbd> is the default hotkey.
-- Search programs, bookmarks and control panel items with acronyms eg. `gk` or `gp` for GitKraken Preview; fuzzy search eg. `code` or `vis` for Visual Studio Code.
-- Open context menu: on the selected result, press <kbd>Ctrl</kbd>+<kbd>O</kbd>/<kbd>Shift</kbd>+<kbd>Enter</kbd>.
+- Open context menu: <kbd>Ctrl</kbd>+<kbd>O</kbd>/<kbd>Shift</kbd>+<kbd>Enter</kbd>.
 - Cancel/Return to previous screen: <kbd>Esc</kbd>.
-- Install/Uninstall/Update plugins: in the search window, type `pm install`/`pm uninstall`/`pm update` + the plugin name.
-- Press `F5` while in the query window to reload all plugin data.
+- Install/Uninstall plugins: in the search window, type `wpm install/uninstall` + the plugin name.
```
Comment (jjw24) [inline README.md:55]: wpm?

**Comment by jjw24** [comment]
moved the tips to https://github.com/Flow-Launcher/docs/pull/3

**Review by jjw24 — DISMISSED** [review]
added a two badges- contributors and Documentation, updated Python install method back to the old way and will pr the new method

**Review by jjw24 — APPROVED** [review]

**Pull request merged**